katrain/i18n/locales/en/LC_MESSAGES/katrain.po

```diff
@@ -553,3 +553,12 @@ msgstr ""
 "Picks moves biased below the `threshold` line and plays the best one. "
 "Increase `line_weight` to penalize moves near the center more. Stops "
 "strategy after the 'endgame' fraction of the board is filled."
+
+msgid "ai:p:rank"
+msgstr "Rank 12k - 3d"
```

Comment:
I think we can pick a better name, especially this might be confusing if we want to calibrate other ais like this